src/components/commons/Footer.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '../../assets/styles/commons/Footer.scss';
+import AFL from '../../assets/images/AFL.svg';
+
+const Footer = () => (
+	<div className="footer-container">
+		<div className="left-footer">
+			<div>
+				<h3>CONTACT US FORM</h3>
+			</div>
+			<div className="email-form">
+				<p>
+					In case you do find the need to contact us, please provide
+					your email and we shall revert as soon as possible.
+				</p>
+				<input defaultValue="Email" />
+				<button type="submit">CONTACT US</button>
+			</div>
+			<div>Copyright © 2019 | All Rights Reserved</div>
+		</div>
+		<div className="center-footer">
+			<div>Standings</div>
```

Comment:
Are these links or? 
Are our users going to interact with them?
If users will interact with them, add them to the tab order by either add a 'tabIndex` prop to the `div` or using a semantic HTML element that is by default in the tab order like a button or an anchor tag